dimod/generators/fcl.py

```diff
@@ -144,6 +155,17 @@ def frustrated_loop(graph: GraphLike,
         raise RuntimeError(f"number of found cycles, {good_cycles}, is below " 
                            f"requested {num_cycles}")
 
+    if planted_solution is not None:
+        warnings.warn("planted_solution is deprecated; use spin-reversal transforms: "
+                      "(1) bqm.flip_variable() for superficial reorientation of "
+                      "spin labels to match desired planting or "
+                      "(2) SpinReversalComposite for presentation of a randomized "
+                      "planted solution to some solver.",
+                      DeprecationWarning)
```

Comment:
need `stacklevel=2` (or maybe `stacklevel=3` because of the decorator) as well so the warning actually tells the user where to change things.